client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -173,6 +173,7 @@ body {
 
   &:hover, &:focus {
     color: @yellow-darker;
+    cursor: pointer;
```

Comment:
@ranbena this adds some work for #4017... Sorry, but @gabrieldutra is right: we can YOLO it, but only after V8 branch is cut.